src/site/content/en/react/virtualize-long-lists-react-window/index.md

```diff
@@ -346,6 +347,39 @@ const Row = ({ index, style }) => {
 };
 ```
 
+## Overscanning
+
+Since items in a virtualized list only change when the user scrolls, blank
+space can briefly flash as newer entries are about to be displayed. You can
+try quckly scrolling any of the previous examples in this guide to notice
+this. 
+
+To improve the user experience of virtualized lists, `react-window` allows
+you to overscan items with the `overscanCount` property. This allows you to
+define how many items outside of the visible "window" to render at all times.
+
+```js
```

Comment:
I just noticed that `jsx` is a supported type in our syntax highlighter. So if you want you can use that throughout?

Reply:
Ooh I didn't know

Updated all relevant snippets to `jsx` 👍